scripts/encode-metacomment/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/reviewdog/reviewdog/service/github"
+)
+
+var fprint = flag.String("fprint", "", "fingerprint")
+var toolName = flag.String("tool-name", "", "tool-name")
+
+func main() {
+	flag.Parse()
+	if fprint == nil || toolName == nil {
```

Comment:
`flag.String` always returns non-nil value, so this condition is always `false`.

```suggestion
	if *fprint == "" || *toolName == "" {
```